bucket/vt-cli.json

```diff
@@ -0,0 +1,32 @@
+{
+    "homepage": "https://github.com/VirusTotal/vt-cli",
+    "description": " VirusTotal CLI, a tool designed for those who love both VirusTotal and command-line interfaces. With this tool you can do everything you'd normally do using the VirusTotal's web page",
+    "version": "0.7.0",
+    "license": "Apache License 2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/VirusTotal/vt-cli/releases/download/0.7.0/Windows64.zip",
+            "hash": "sha256:2468803342F0E12853D2761D14AAFE2E9EBC086CBF1C7F9CC334F430BB1C4167"
+        },
+        "32bit": {
+            "url": "https://github.com/VirusTotal/vt-cli/releases/download/0.7.0/Windows32.zip",
+            "hash": "sha256:C453A6AF6C86301FACF7FEECCC469DBC91901C17911F75E95197CE526650CE7B"
+        }
+    },
+    "bin": [
+        "vt.exe"
+    ],
+        "checkver": {
+        "github": "https://github.com/VirusTotal/vt-cli/"
+    },
```

Comment:
```suggestion
    "bin": "vt.exe",
    "checkver": "github",
```

Reply:
I tried to accept all your suggestions in github. Not sure if I did it correctly though